pkg/kyma/cmd/install/cmd.go

```diff
@@ -72,14 +72,42 @@ func NewCmd(o *Options) *cobra.Command {
 	cobraCmd := &cobra.Command{
 		Use:   "install",
 		Short: "Installs Kyma on a running Kubernetes cluster",
-		Long: `Install Kyma on a running Kubernetes cluster.
-
-Make sure that your KUBECONFIG is already pointing to the target cluster.
-The command:
-- Installs Tiller
-- Deploys the Kyma Installer
-- Configures the Kyma Installer using the latest minimal configuration
-- Triggers Kyma installation
+		Long: `Installs Kyma on a running Kubernetes cluster.
+
+Prerequisites:
+- Kyma is not installed.
+- Kubernetes cluster is available with your KUBECONFIG already pointing to it.
+
+Standard installation steps:
+1. Fetch the tiller.yaml file from /installation/resources directory and deploy it to the cluster.
```

Comment:
I'm imagining reading that as inline help. It describes to me what the CLI will do when calling the command. Shouldn't it be in a different ... grammar, like "Fetches" and "Deploys"